front/src/applications/operationalStudies/components/MacroEditor/ngeToOsrd.ts

```diff
@@ -286,12 +286,12 @@ const handleUpdateTrainSchedule = async ({
   const { nodes, labels } = netzgrafikDto;
   const trainrunSectionsByTrainrunId = getTrainrunSectionsByTrainrunId(netzgrafikDto, trainrun.id);
   // TODO Paced train : Adapt to handle paced trains in issue https://github.com/OpenRailAssociation/osrd/issues/10612
-  const formattedTrainrunIdToUpdate = formatTrainScheduleIdToEditoastTrainId(
+  const formattedTrainScheduleIdToUpdate = formatTrainScheduleIdToEditoastTrainId(
```

Comment:
nit (editoastIdToUpdate may be enough)
```suggestion
  const editoastTrainScheduleIdToUpdate = formatTrainScheduleIdToEditoastTrainId(
```

Reply:
Done